src/Microsoft.Azure.SignalR.Protocols/ServiceMessage.cs

```diff
@@ -23,6 +22,23 @@ public class HandshakeRequestMessage : ServiceMessage
         /// </summary>
         public int Version { get; set; }
 
+        /// <summary>
+        /// Gets or sets the type of service connection.
+        /// </summary>
+        /// <value>
+        /// <list type="bullet">
+        /// <item>0, Default, it can carry clients, service runtime should always accept this kind of connection.</item>
+        /// <item>1, OnDemand, it can carry clients, but it can be rejected by service runtime.</item>
```

Comment:
Add more details? e.g. service side requests server side to initiate the connection during runtime through ping message status sync?